nautobot/ui/src/components/ObjectDetail/RenderCol.js

```diff
@@ -0,0 +1,25 @@
+import { NautobotGrid, NautobotGridItem } from "@nautobot/nautobot-ui";
+
+import RenderGroup from "./RenderGroup";
+
+function RenderCol({ tabData, data, schema }) {
```

Comment:
I think many of these are good enough to be used generically. Maybe `components/render` instead of `components/ObjectDetail`?